docs/changelog.md

```diff
@@ -12,6 +12,10 @@
 
 ### Version 1.3.2
 
+* Added `storage_type` operator and `storage_type()` function [#781][], with pending deprecation for _storage_type [#786][] [#790][]
+* Better errors generated for missing or incorrect sample to mean storage [#782][]
+* Explicit reset() documentation [#783][]
```

Comment:
```suggestion
* Explicit `reset()` documentation [#783][]
```

Reply:
Though I'm okay to not mention docs improvements, at least specifically. It's fine though. :)

Reply:
Yeah I thought about that too, but then I thought we can add it to the issue to close it completely. I just realized we already closed it before with a previous PR merge (https://github.com/scikit-hep/boost-histogram/issues/567)